### PR TITLE
fix: restores display helpers with relevant mixins

### PR DESCRIPTION
**Pull request by eboyer**
Closes sap/fundamental#1296

### CODE REVIEW

**Comment by joseegm** [comment]
Test Version for this PR was deployed

Built with commit 7725081e717316263d3f5788edf98cf1bf0cc140

https://deploy-preview-1357--fundamental.netlify.com

File: scss/_mixins.scss [hunk 1]
```diff
@@ -4,3 +4,4 @@
 @import "./mixins/states";
 @import "./mixins/type";
 @import "./mixins/vars";
+@import "./mixins/breakpoints";
```
Comment (bcullman) [inline scss/_mixins.scss]: breakpoints is unrelated. either remove, or retitle PR

Comment (eboyer) [reply scss/_mixins.scss]: @bcullman the breakpoints file is utilized on line 14 of `_display.scss` with the `breakpoint-infix` function.

Comment (eboyer) [reply scss/_mixins.scss]: @bcullman ~ so they come hand in hand. they were deleted together.

Comment (droshev) [reply scss/_mixins.scss]: then maybe the PR should be generalize of reverting removed classes and mixin

Comment (eboyer) [reply scss/_mixins.scss]: @droshev @bcullman ~ changed title of PR to `fix: restores display helpers with relevant mixins`

**Pull request merged**